.github/workflows/build.yml

```diff
@@ -64,6 +64,19 @@ jobs:
       - name: Package
         run: npm run build
 
+      - name: Release
+        if: |
+          startsWith(matrix.os, 'ubuntu') &&
+          github.repository == 'denoland/vscode_deno' &&
+          startsWith(github.ref, 'refs/tags/')
+        uses: softprops/action-gh-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          files: |
+            vscode-deno.vsix
+          draft: true
```

Comment:
Maybe turn off the draft? I just use this on the deno release to update the release notes before publication.